src/test/resources/testng.xml

```diff
@@ -3,12 +3,12 @@
 <suite name="Suite1" verbose="1" >
     <test name="PlaceLab Login test" >
         <classes>
-            <class name="com.placelab.aldinrizvo.tests.SmokeTest" />
             <class name="com.placelab.aldinrizvo.tests.ValidLoginTest" />
             <class name="com.placelab.aldinrizvo.tests.InvalidEmailLoginTest" />
             <class name="com.placelab.aldinrizvo.tests.InvalidPasswordLoginTest" />
-            <class name="com.placelab.aldinrizvo.tests.ValidLoginTestByPressingEnterKey" />
+            <class name="com.placelab.aldinrizvo.tests.ValidLoginByPressingEnterKeyTest" />
             <class name="com.placelab.aldinrizvo.tests.RestorePasswordTest" />
+            <class name="com.placelab.aldinrizvo.tests.CreateSinglePlaceSearchReportSmokeTest" />
```

Comment:
report related tests do not belong among login tests, you can make a separate test tag for them in this file